backend/db/index.js

```diff
@@ -26,8 +26,7 @@ const findOneFromDB = (findCondition, collectionName) => {
 };
 
 const updateOneFromDb = async (condition, data, collectionName) => {
-  const res = await db.collection(collectionName).updateOne(condition, { $set: data });
-  return res;
+  return await db.collection(collectionName).updateOne(condition, { $set: data });
 };
```

Comment:
We don't even need async/await here. It can be just :
```
const updateOneFromDb = (condition, data, collectionName) => db.collection(collectionName).updateOne(condition, { $set: data })
```